mis_builder/static/src/components/mis_report_widget.xml

```diff
@@ -16,6 +16,11 @@
                                 <FilterMenu t-if="showSearchBar" />
                             </div>
                             <div class="oe_mis_builder_action_buttons">
+                                <DatePicker
+                                    date="state.date"
+                                    onDateTimeChanged="onDateTimeChanged.bind(this)"
+                                    t-if="comparison_mode"
```

Comment:
I think we need a new `widget_show_pivot_date` flag to configure the visibility, because there are some report where the pivot date is not used at all.

Reply:
Well, I used comparison mode, because the field is only shown on the form if comparison_mode is true

:thinking:

Anyway, I make the change